pkg/jobs/jobs.go

```diff
@@ -172,7 +176,7 @@ func deprecatedIsOldSchemaChangeJob(payload *jobspb.Payload) bool {
 // Terminal returns whether this status represents a "terminal" state: a state
 // after which the job should never be updated again.
 func (s Status) Terminal() bool {
-	return s == StatusFailed || s == StatusSucceeded || s == StatusCanceled
+	return s == StatusFailed || s == StatusSucceeded || s == StatusCanceled || s == StatusRevertFailed
```

Comment:
Given this job state is treated as `Terminal`, should they also be considered a candidate for GC?

Reply:
A major motivation here is to not GC these things.

Reply:
Ah, I had incorrectly thought that we check if a job is Terminal to determine if it's a candidate for GC, but that's not the case. Sorry for the noise.